config/env_sample.json

```diff
@@ -121,9 +121,9 @@
     "/*  DATA_WAREHOUSE_PORT=5432": "*/",
     "/*  Enable/Disable Unizin Date Warehouse specific features/data": "*/",
     "DATA_WAREHOUSE_IS_UNIZIN": true,
-    "RESOURCE_TYPES": {
-        "files": {"resource_label" :"Files", "resource_value":"files", "resources": ["canvas"], "disabled": "false"},
-        "videos": {"resource_label":"Videos", "resource_value":"videos", "resources": ["leccap"], "disabled": "false"}
+    "RESOURCE_VALUES": {
+        "files":  ["canvas"],
+        "videos": ["leccap"]
```

Comment:
@vjcao This structure should work and is simplified as well

Reply:
I think this will still need the labels somewhere. Though the better way to do that would be in an i18n message file.

Reply:
why do we need the labels? what's the purpose?

Reply:
Because it's intended that these will be dynamically inserted in the UI like if we added a new reading configuration.

We could possibly use the keys as the labels on the page. (if we uppercased the keys)

Reply:
Currently, the API sends something like `['Files', 'Video'] based on the `RESOURCE_VALUES` mapping. I get your point, but I am sending what UI needs they can just grab. I think if we pose a case sensitive property the next person who comes in adds a new value if not aware the casing thing it can be easily missed.

Reply:
css can text-transform: capitalize. 

But as soon as I get some time the things I'd love to focus on are i18n and DRF.

Reply:
So you want it property as below
```
"RESOURCE_VALUES": {
        "Files":  ["canvas"],
        "Videos": ["leccap"]
    },
```

Reply:
That or the CSS text-transform class would be the easy quick fix for this for now. Then it could still be dynamic in the UI.